package-lock.json

```diff
@@ -1,8 +1,12479 @@
 {
   "name": "fe-course-basic-2023",
   "version": "21.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
here is the change

Reply:
In this task you should only add your Readme, not change the settings of whole repo.
So yes, if you change 2 to 1 back, it returns the file to initial state and remove its changes from your pull-request.

Reply:
the most interesting thing is that I didn't touch that file, I don't even know why there are changes)

Reply:
did i do my homework ?

Reply:
Hi, there, sorry, but not. You still have some changes for the project's package-lock.json. If I merge your pr, these changes will rewrite it. And that is not, what you have to do in this task.
I've wrote a way to deal with this problem and its explanation in tg chat - https://t.me/c/1602765599/177.
Or you could ask Bard/ChatGPT what's going on and how to solve it